### ISSUE TITLE
UML Class Diagram omitting class members

### ISSUE THREAD

**Issue by naijie2108** [nus-cs2113-AY2122S1/forum#159]
While it's allowed to omit the 'Operations' compartment and/or the 'Attributes' compartment, is it okay to omit only certain methods/attributes? Like for example, for a child class, is it okay to omit the inherited methods? Thank you

**Comment by okkhoy** [comment]
Yes. You are allowed to omit details that are not important to understand the context of the discussion.
